README.md

```diff
@@ -275,15 +275,25 @@ random_tiles_extractor = RandomTiler(
 ```
 
 Notice that we also specify the random seed to ensure the
-reproducibility of the extraction process. Starting the extraction is as
-simple as calling the `extract` method on the extractor, passing the
+reproducibility of the extraction process.
+
+We may want to check which tiles have been selected by the tiler, before starting the extraction procedure and save them;
+the ``locate_tiles`` method of ``RandomTiler`` returns the thumbnail of the slide with the corresponding tiles outlined.
+
+```python
+random_tiles_extractor.locate_tiles(slide=prostate_slide)
```

Comment:
i'd like to see a complete usage example of the locate_tiles using all the available parameters